### DISCUSSION TITLE
GridEditController not exported from @serenity-is/pro.extensions?

### DISCUSSION THREAD

**Discussion by LSNicholls** [serenity-is/Serenity#6582] [Help]
Hi folks,

I am not putting this in as an issue because there is every possibility that I am just being a stupid newbie and there is no bug.  BUT, I have seen this consistently no matter how untouched my project is, and even when everything else is referenced and builds properly:

If I use the following, per the inline editing sample:

`import { GridEditController } from '@serenity-is/pro.extensions';`

... I consistently get an error in Visual Studio 's error list:

`Error	TS2305	(TS) Module '"@serenity-is/pro.extensions"' has no exported member 'GridEditController'.`	

The wierd thing is that the build reports no errors and the code that uses the GridEditController class sometimes works fine, if all the other stars aline.  It's just in the Error List.

Does this happen to anybody else??

**Comment by LSNicholls** [comment]
PS actually it is not working fine, it is no longer handling saves right.  I mean, I can't get the save button to appear any more.  

I am now using a *very* simple table, no lookups or anything.  The undo button shows up, but it's not very useful since the changes never take. I tried autoSave:true,  as an editorParams value, no dice.

Ugh.  This *was* working earlier, and the only clue I have is that error that I have supplied here, I don't know what else is changed.

The sad thing is this inline editing sample (on the product table in NorthWind) was the reason I wanted to use Serenity in the first place.  And when it worked earlier I was so happy!  

This is exactly the experience that we need to provide, especially if autoSave is supposed to work. (IOW, as Excel-like an editing experience as possible without a lot of fuss.)

**Comment by LSNicholls** [comment]
Hah!  **It is not me.**  I started a brand new project with all the samples and I updated the Serenity packages for the new project.  And guess what?  It doesn't work there either.
![image](https://user-images.githubusercontent.com/88156108/202067215-3577ee8b-297c-4e5b-a1d3-2e1f22849a5c.png)

**Comment by VictorTomaili** [comment]
Hello @LSNicholls 
Please use StartSharp issues to ask pro related questions. 
Thank you for understanding.

**Comment by LSNicholls** [comment]
I will thank you. Notice that I did, when I escalated it as an issue.  But I didn’t see a discussions facility in StartSharp and really didn’t want to make this an issue initially.

 

>L<

Lisa Slater Nicholls

**Reply by VictorTomaili** [reply]
In StartSharp, issues are the place for discussions 👍
